<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="12" passed="8" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-13T11:21:59 IST" name="ProviderPlatformSuite" finished-at="2025-06-13T11:22:29 IST" duration-ms="30226">
    <groups>
    </groups>
    <test started-at="2025-06-13T11:21:59 IST" name="AllTests" finished-at="2025-06-13T11:22:29 IST" duration-ms="30226">
      <class name="tests.LoginTest">
        <test-method is-config="true" signature="BrowserMethod()[pri:0, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:21:59 IST" name="BrowserMethod" finished-at="2025-06-13T11:22:00 IST" duration-ms="1282" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowserMethod -->
        <test-method signature="logintest()[pri:1, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:22:00 IST" name="logintest" description="Login with valid credentials" finished-at="2025-06-13T11:22:03 IST" duration-ms="2456" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
        <test-method signature="loginWithInvalidCredentials()[pri:2, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:22:03 IST" name="loginWithInvalidCredentials" description="Login with Invalid credentials" finished-at="2025-06-13T11:22:05 IST" duration-ms="1781" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithInvalidCredentials -->
        <test-method signature="loginWithBlankUsername()[pri:3, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:22:05 IST" name="loginWithBlankUsername" description="Login without username" finished-at="2025-06-13T11:22:07 IST" duration-ms="2779" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithBlankUsername -->
        <test-method signature="loginWithBlankPassword()[pri:4, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:22:07 IST" name="loginWithBlankPassword" description="Login without password" finished-at="2025-06-13T11:22:09 IST" duration-ms="1527" status="FAIL">
          <exception class="org.openqa.selenium.NoAlertPresentException">
            <message>
              <![CDATA[no such alert
  (Session info: chrome=137.0.7151.69)
Build info: version: '4.27.0', revision: 'd6e718d134'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [960815b0367a73655382b307da9624d4, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.70 (dfa4dc56b2ab..., userDataDir: C:\Users\Admin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50185}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50185/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 960815b0367a73655382b307da9624d4]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoAlertPresentException: no such alert
  (Session info: chrome=137.0.7151.69)
Build info: version: '4.27.0', revision: 'd6e718d134'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [960815b0367a73655382b307da9624d4, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.70 (dfa4dc56b2ab..., userDataDir: C:\Users\Admin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50185}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50185/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 960815b0367a73655382b307da9624d4
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:618)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:622)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.alert(RemoteWebDriver.java:1194)
at tests.LoginTest.loginWithBlankPassword(LoginTest.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoAlertPresentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithBlankPassword -->
        <test-method signature="loginWithBlankUsernameAndPassword()[pri:5, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:22:09 IST" name="loginWithBlankUsernameAndPassword" description="Login with blank username and password" finished-at="2025-06-13T11:22:10 IST" duration-ms="665" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithBlankUsernameAndPassword -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginTest@9d157ff]" started-at="2025-06-13T11:22:10 IST" name="tearDown" finished-at="2025-06-13T11:22:10 IST" duration-ms="695" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.patientForm">
        <test-method is-config="true" signature="BrowserMethod()[pri:0, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:10 IST" name="BrowserMethod" finished-at="2025-06-13T11:22:13 IST" duration-ms="2531" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowserMethod -->
        <test-method is-config="true" signature="runLogin()[pri:0, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:13 IST" name="runLogin" finished-at="2025-06-13T11:22:14 IST" duration-ms="1476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runLogin -->
        <test-method signature="addPatient()[pri:1, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:14 IST" name="addPatient" description="Fill all the fields with valid data" finished-at="2025-06-13T11:22:17 IST" duration-ms="2189" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addPatient -->
        <test-method signature="validateTableEntry()[pri:2, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:17 IST" name="validateTableEntry" description="When submit the form, Table should increased by one" finished-at="2025-06-13T11:22:18 IST" duration-ms="1827" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTableEntry -->
        <test-method signature="clickSubmit()[pri:3, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:18 IST" name="clickSubmit" description="Leave all the fields empty and click submit" finished-at="2025-06-13T11:22:19 IST" duration-ms="469" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickSubmit -->
        <test-method signature="invalidDOB()[pri:4, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:19 IST" name="invalidDOB" description="Enter Invalid DOB formate" finished-at="2025-06-13T11:22:21 IST" duration-ms="1974" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[entry not appear in table expected [4] but found [5]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: entry not appear in table expected [4] but found [5]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at tests.patientForm.invalidDOB(patientForm.java:102)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidDOB -->
        <test-method signature="testFutureDOB()[pri:5, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:21 IST" name="testFutureDOB" description="Future DOB entry" finished-at="2025-06-13T11:22:23 IST" duration-ms="2000" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at tests.patientForm.testFutureDOB(patientForm.java:124)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFutureDOB -->
        <test-method signature="testSpecialCharInMRN()[pri:6, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:23 IST" name="testSpecialCharInMRN" description="Special characters in MRN" finished-at="2025-06-13T11:22:25 IST" duration-ms="2087" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[entry not appear in table expected [6] but found [7]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: entry not appear in table expected [6] but found [7]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at tests.patientForm.testSpecialCharInMRN(patientForm.java:147)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSpecialCharInMRN -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.patientForm@6e57e95e]" started-at="2025-06-13T11:22:25 IST" name="tearDown" finished-at="2025-06-13T11:22:25 IST" duration-ms="361" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.patientForm -->
      <class name="tests.LogoutTest">
        <test-method is-config="true" signature="BrowserMethod()[pri:0, instance:tests.LogoutTest@3676ac27]" started-at="2025-06-13T11:22:25 IST" name="BrowserMethod" finished-at="2025-06-13T11:22:26 IST" duration-ms="1228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowserMethod -->
        <test-method signature="testSpecialCharInMRN()[pri:6, instance:tests.LogoutTest@3676ac27]" started-at="2025-06-13T11:22:26 IST" name="testSpecialCharInMRN" description="Special characters in MRN" finished-at="2025-06-13T11:22:29 IST" duration-ms="2306" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSpecialCharInMRN -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LogoutTest@3676ac27]" started-at="2025-06-13T11:22:29 IST" name="tearDown" finished-at="2025-06-13T11:22:29 IST" duration-ms="307" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LogoutTest -->
    </test> <!-- AllTests -->
  </suite> <!-- ProviderPlatformSuite -->
</testng-results>
